src/wcs1_0/src/main/java/applicationContext.xml

```diff
@@ -94,6 +94,11 @@
     <constructor-arg ref="geoServer"/>
   </bean>
 
+  <bean id="wcs100ElevationKvpParser" class="org.geoserver.wcs.kvp.ElevationKvpParser">
```

Comment:
Won't this conflict with the "elevation" KVP parser in WMS? Best to limit it to the WCS service, and eventually to the version 1.0 version too.

Reply:
I wasn't sure what to do about that but the conflict already exists with the WCS and WMS TimeKvpParser.  I'm wondering if that should be a separate PR that limits the WCS and WMS TimeKvpParser and ElevationKvpParser.

Reply:
It the PR is done promptly I see no problem, if you're not sure about doing it, I'd avoid introducing yet another potential conflict.

Reply:
I just went ahead and fixed the WCS ElevationKvpParser.